src/transformation/visitors/expression-statement.ts

```diff
@@ -4,8 +4,14 @@ import { FunctionVisitor } from "../context";
 import { transformBinaryExpressionStatement } from "./binary-expression";
 import { transformLuaTableExpressionStatement } from "./lua-table";
 import { transformUnaryExpressionStatement } from "./unary-expression";
+import { transformMultiDestructuringAssignmentStatement } from "./language-extensions/multi";
 
 export const transformExpressionStatement: FunctionVisitor<ts.ExpressionStatement> = (node, context) => {
+    const multiResult = transformMultiDestructuringAssignmentStatement(context, node);
```

Comment:
Could you check if this is a multireturn first before calling any transformation functions? We had a big performance issue a while ago where we did a lot more work than we needed because we were calling transformation functions too early. is
```ts
if (isMultiDestructuringAssignment(context, node)) {
    return transformMultiDestructuringAssignmentStatement(context, node);
}
```